oak_runtime/src/introspection_browser_client/index.tsx

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import introspectionEventsProto from './proto/introspection_events_pb';
+
+function loadSerializedEvents(): Promise<Uint8Array> {
```

Comment:
Perhaps add a comment explaining what it does. In particular, it might be nice to give a bit of context about what the `Events` are.